editor/debugger/script_editor_debugger.cpp

```diff
@@ -342,7 +342,7 @@ void ScriptEditorDebugger::_msg_debug_enter(uint64_t p_thread_id, const Array &p
 	const Thread::ID caller_id = p_data[3];
 
 	ThreadDebugged td;
-	td.name = (caller_id == Thread::get_main_id()) ? TTR("Main Thread") : itos(caller_id);
+	td.name = (caller_id == Thread::get_main_id()) ? TTRC("Main Thread") : itos(caller_id);
```

Comment:
The `OptionButton` showing this info has auto translation disabled. We can enable that and ignore the fact that regular thread ID strings will be unnecessarily translated.

A better way would be changing `ThreadDebugged::name` to a thread ID and decide the display name when populating the `OptionButton`. Besides, `ThreadDebugged` also has a `thread_id` and I'm not sure if it's the same as `caller_id`. I think this improvement could be a separate PR.

Reply:
The list seems to be refreshed on every debugger break, so it's probably fine to leave TTR here.